src/dev-app/system-config.ts

```diff
@@ -15,11 +15,13 @@ System.config({
     'node:*': 'node_modules/*'
   },
   map: {
-    'rxjs': 'node:rxjs',
     'main': 'main.js',
     'tslib': 'node:tslib/tslib.js',
     'moment': 'node:moment/min/moment-with-locales.min.js',
 
+    'rxjs': 'node_modules/rxjs/bundles/rxjs.umd.min.js',
```

Comment:
Should we serve the unminified files so we can get better stack traces? File size doesn't really matter locally.

Reply:
Usually there is no helpful RxJS stack (see https://github.com/ReactiveX/rxjs/issues/2663), and you can always enable source maps. 

Also, I might switch the Angular stuff to minified as well, but that's another story and I need to make sure that it still works w/ source-maps etc.